working_with_the_llvm_ast.md

```diff
@@ -64,42 +74,42 @@ binary using the clang-tidy script to observe whether it matches any violations
 violations. 
 
 
-### Using clang-query on Compiler Explorer v/s Using it in your Terminal
+### Using clang-query on Compiler Explorer vs Using it in your Terminal
 
 It is highly advisable to do all clang-query experimentation on [Compiler Explorer](www.godbolt.org). The clang-query
 command line tool as of now has very poor formatting capability. For example, the following matcher matches any calls
-made to `std::move` - 
+made to `std::move()`:
 
 ```
```

Comment:
nitpick: A few of the examples are still missing the language specifier. If there is no language, txt will work too.